apps/framework-cli/src/framework/core/plan.rs

```diff
@@ -44,8 +44,8 @@ pub async fn plan_changes(
     client: &mut ClientHandle,
     project: &Project,
 ) -> Result<InfraPlan, PlanningError> {
-    let mut target_infra_map = if project.is_production {
-        let json_path = Path::new(".moose/infrastructure_map.json");
+    let json_path = Path::new(".moose/infrastructure_map.json");
+    let mut target_infra_map = if project.is_production && json_path.exists() {
```

Comment:
this is for running prod locally i guess? i worry that this might hide a bug

Reply:
Yes that's the reason, I mean we could add a log if you want.

Reply:
probably add an "assert i'm not inside docker" if `is prod` && `json not exists`